perllib/FixMyStreet/Cobrand/TfL.pm

```diff
@@ -0,0 +1,33 @@
+package FixMyStreet::Cobrand::TfL;
+use parent 'FixMyStreet::Cobrand::UKCouncils';
```

Comment:
Want to make this Whitelabel, then can drop the empty front/footer-marketing.html template from this commit.